modules/ROOT/pages/auradb/getting-started/create-database.adoc

```diff
@@ -34,21 +28,24 @@ You can only create one AuraDB Free instance per account.
 [.include-with-AuraDB-Professional]
 =====
 
+[NOTE]
+======
+If you have yet to create a free instance, you will always be presented with the option to do so. If you still wish to create a professional instance, select *From $65 / month* and follow the instructions below from step 3 onwards.
```

Comment:
*From $65 / month*

Is this going to be a fixed string?

Reply:
I am just worried that the offering might change and this would become outdated. Can we rephrase it somehow?

Reply:
Unfortunately I am limited by the name of the button, i did give some feedback about changing the name of the button but its not changing for the release.

Reply:
This is actually being changed after our feedback, update to come